doc/FLOW.md

```diff
@@ -11,8 +11,11 @@ It can be seen as:
 
 # Installation #
 
-Start by following the standard ivre installation.
-
+Start by following the standard ivre installation. Don't forget to install 
+py2neo package
```

Comment:
I'd be OK to make it clear here that py2neo (and neo4j, actually) is required. However, I would not recommend one particular method to install it over others. Could you remove the script (and mention the installation of a Neo4j server)?